sdk/formrecognizer/ai-form-recognizer/README.md

```diff
@@ -134,6 +134,12 @@ to illustrate using long-running operations [below](#Examples).
 ## Examples
 The following section provides several JavaScript code snippets illustrating common patterns used in the Form Recognizer client libraries.
 
+* [Recognize receipts](#recognize-receipts "Recognize receipts")
+* [Recognize content](#recognize-content "Recognize content")
+* [Train model](#train-model "Train model")
+* [Recognize forms using a custom model](#recognize-forms-using-a-custom-model "Recognize forms using a custom model")
+* [Listing all models](#listing-all-models "Listing all models")
```

Comment:
I am guessing the last part in the quotes is for the tooltip content to show when hovering over the link?
Tooltips with the exact same context as the link text do not add any value.
So, unless you have any extra information to provide that is not available in the link text, you should not need to have the tooltips here

```suggestion
* [Recognize receipts](#recognize-receipts)
* [Recognize content](#recognize-content)
* [Train model](#train-model)
* [Recognize forms using a custom model](#recognize-forms-using-a-custom-model)
* [Listing all models](#listing-all-models)
```